File: MPI/mpicheck/tests/functions/3.0/MPI_Get_accumulate.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       INTEGER var_1
       INTEGER var_2
       TYPE(INTEGER) var_3(10)
       INTEGER var_4
       INTEGER var_5
       INTEGER var_6
       INTEGER(KIND=MPI_ADDRESS_KIND) var_7
       INTEGER var_8
       INTEGER var_9
       INTEGER var_10
       INTEGER var_11
       INTEGER var_12
        call mpi_get_accumulate(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10, var_11, var_12)
       call pmpi_get_accumulate(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10, var_11, var_12)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Get_accumulate.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    int var_1;
    MPI_Datatype var_2;
    void *var_3;
    int var_4;
    MPI_Datatype var_5;
    int var_6;
    MPI_Aint var_7;
    int var_8;
    MPI_Datatype var_9;
    MPI_Op var_10;
    MPI_Win var_11;
    int ret;
    /* calls */
    ret = MPI_Get_accumulate(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9, var_10, var_11);
    ret = PMPI_Get_accumulate(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9, var_10, var_11);
    return 0;
}
